plugins/modules/rds_cluster.py

```diff
@@ -1261,6 +1338,12 @@ def main():
     else:
         cluster_id = module.params["db_cluster_identifier"]
 
+    if cluster and module.params.get("remove_from_global_db"):
+        if cluster["Engine"] in ["aurora", "aurora-mysql", "aurora-postgresql"]:
+            if changed:
+                wait_for_cluster_status(client, module, cluster_id, "cluster_available")
```

Comment:
This line will throw `botocore.exceptions.WaiterError: Waiter cluster_available failed: Max attempts exceeded. Previously accepted state: Matched expected service error code: DBClusterNotFoundFault` when state is absent and remove_from_global_db is true.

Reply:
I don't think that should happen, as the call to `remove from global db` takes place before it makes call to `delete` the cluster

Reply:
I had added these as suggested by @alinabuzachis

Reply:
@GomathiselviS made the fix, should work now